apps/aeutils/src/aeu_requests.erl

```diff
@@ -142,29 +143,57 @@ get_header_by_hash(Uri, Hash) ->
             {error, unexpected_response}
     end.
 
+-spec get_n_hashes(http_uri_uri(),  binary(), non_neg_integer()) -> response([{integer(), binary()}]).
+get_n_hashes(Uri, Hash, N) when is_integer(N) ->
+    EncHash = aec_base58c:encode(block_hash, Hash),
+    Response = process_request(Uri, 'GetHeadersByHash', [{"hash", EncHash}, {"number", integer_to_list(N)}]),
+    case Response of
+        {ok, 200, Data} when is_list(Data) ->
+            %% Keep them in order, oldest block is first!
+            {ok, lists:foldr(fun(Header, Acc) ->
+                            case aec_headers:deserialize_from_map(Header) of
+                                {ok, H} ->
+                                    {ok, HH} = aec_headers:hash_header(H),
+                                    [ {aec_headers:height(H), HH} | Acc ];
+                                 _ ->
+                                    lager:info("Got bad block hash from ~p", [Uri]),
+                                    Acc
+                            end
+                        end, [], Data)};
+        {error, _Reason} = Error ->
+            Error;
+        _ ->
+            %% Should have been turned to {error, _} by swagger validation
+            lager:debug("unexpected response (~p): ~p", [Uri, Response]),
+            {error, unexpected_response}
+    end.
 
 %% Add API for header later... now use block
 -spec get_header_by_height(http_uri_uri(), non_neg_integer()) -> response(aec_headers:header()).
 get_header_by_height(Uri, Height) when is_integer(Height) ->
-    Response = process_request(Uri, 'GetBlockByHeight', [{"height", integer_to_list(Height)}]),
+    Response = process_request(Uri, 'GetHeaderByHeight', [{"height", integer_to_list(Height)}]),
```

Comment:
I see this means that this node cannot sync from node running v0.8.0 node. Could you please propose a line in the [releases notes](https://github.com/aeternity/epoch/blob/32722bae6d04dfcaf4b50d8e19a2eca5fdbded78/docs/release-notes/RELEASE-NOTES-0.9.0.md#about-this-release)?

Reply:
Anyway the consensus changed, though a line may be beneficial.